bump version, fix import test

agcopenhaver opened the pull request:
<!--
Thank you for pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

Commit:
bump version, fix test import

agcopenhaver commented:
@conda-forge-admin, please rerender

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

conda-forge-admin committed:
MNT: Re-rendered with conda-build 3.17.8, conda-smithy 3.4.0, and conda-forge-pinning 2019.07.04

Commit:
make noarch

agcopenhaver commented:
@conda-forge-admin, please rerender

conda-forge-admin committed:
MNT: Re-rendered with conda-build 3.17.8, conda-smithy 3.4.0, and conda-forge-pinning 2019.07.04

Commit:
pypi to github

it's tough to know where the official releases are with this repo. Pypi seems more up to date, but download links do not work and I'm not sure if it's the official release. Additionally, it changes the pkg name

agcopenhaver commented:
@pmlandwehr is this good to merge?

Commit:
noarch?

agcopenhaver commented:
@conda-forge-admin, please rerender

conda-forge-admin committed:
MNT: Re-rendered with conda-build 3.17.8, conda-smithy 3.4.0, and conda-forge-pinning 2019.07.04

Commit:
Update meta.yaml

Commit:
Update meta.yaml

Commit:
update sha

Commit:
Update meta.yaml

agcopenhaver commented:
@conda-forge-admin, please rerender

conda-forge-admin committed:
MNT: Re-rendered with conda-build 3.17.8, conda-smithy 3.4.0, and conda-forge-pinning 2019.07.04

Commit:
Update meta.yaml

agcopenhaver commented:
@conda-forge-admin, please rerender

conda-forge-admin committed:
MNT: Re-rendered with conda-build 3.17.8, conda-smithy 3.4.0, and conda-forge-pinning 2019.07.04

agcopenhaver commented:
this doesn't build in windows py35 or greater because of encoding issues in the source setup.py file.

I proposed a change to the source repo, which fixes it. We'll see what they do about it.

Commit:
Update meta.yaml

agcopenhaver commented:
@conda-forge/core I don't know if it's too soon to ping you on this, but it doesn't look like this repo gets a ton of attention

mariusvniekerk commented:
@agcopenhaver feel free to add yourself as a maintainer.

mariusvniekerk commented on recipe/meta.yaml:
```diff
@@ -1,40 +1,46 @@
+  home: "https://github.com/mattgwwalker/msg-extractor"
+  license: "GPL"
+  license_family: "GPL"
+  license_file: ""
```
This has to be filled.   Not distributing the license for GPL packages is a problem.

agcopenhaver replied:
Wow - that was dumb of me. Sorry for the sloppiness.

Commit:
Update meta.yaml

scopatz commented:
@conda-forge-admin please rerender

conda-forge-admin committed:
MNT: Re-rendered with conda-build 3.17.8, conda-smithy 3.4.0, and conda-forge-pinning 2019.07.10
